pkg/drawio/layout.go

```diff
@@ -138,33 +138,36 @@ func sortGroupSquareBySize(groups []SquareTreeNodeInterface) []SquareTreeNodeInt
 //	         - else its visibility is connectedPoint
 func (ly *layoutS) calcGroupsVisibility(subnet SquareTreeNodeInterface) {
```

Comment:
```suggestion
func calcGroupsVisibility(subnet SquareTreeNodeInterface) {
```
Function can be static

Reply:
there are some methods in layout that has the receiver (ly *layoutS) but do not use it.
conceptually they belong to layoutS, static methods.
is there another way to say that they are part of layoutS?

Reply:
done